packages/amplify_datastore/ios/Classes/SwiftAmplifyDataStorePlugin.swift

```diff
@@ -86,7 +89,8 @@ public class SwiftAmplifyDataStorePlugin: NSObject, FlutterPlugin {
     
     private func onConfigureDataStore(args: [String: Any], result: @escaping FlutterResult) {
 
-        guard let modelSchemaList = args["modelSchemas"] as? [[String: Any]],
+        guard let serializedModelSchemas = args["modelSchemas"] as? [[String: Any]],
+              let serializedCustomTypeSchemas = (args["customTypeSchemas"] ?? []) as? [[String: Any]],
```

Comment:
What accounts for the difference here? Do we guarantee at least an empty array on dart side for just model, or could the following suggestion work now?
```suggestion
        guard let serializedModelSchemas = args["modelSchemas"] as? [[String: Any]],
              let serializedCustomTypeSchemas = args["customTypeSchemas"] as? [[String: Any]],
```